test/services/import/legacy/process-licence.service.test.js

```diff
@@ -13,7 +13,7 @@ const { generateUUID } = require('../../../../app/lib/general.lib.js')
 const { generateLicenceRef } = require('../../../support/helpers/licence.helper.js')
 
 // Things to stub
-const PersistLicenceService = require('../../../../app/services/import/persist-licence.service.js')
+const PersistLicenceService = require('../../../../app/services/import/persist-import.service.js')
```

Comment:
Thing being stubbed needs to match name of module being imported.

```suggestion
const PersistImportService = require('../../../../app/services/import/persist-import.service.js')
```